src/lazy/text/buffer.rs

```diff
@@ -705,19 +731,40 @@ impl<'data> TextBufferView<'data> {
             map(
                 match_and_length(Self::match_list),
                 |(matched_list, length)| {
-                    EncodedTextValue::new(MatchedValue::List, matched_list.offset(), length)
+                    // TODO: Cache child expressions found in 1.0 list
+                    let not_yet_used_in_1_0 =
+                        bumpalo::collections::Vec::new_in(self.allocator).into_bump_slice();
+                    EncodedTextValue::new(
+                        MatchedValue::List(not_yet_used_in_1_0),
+                        matched_list.offset(),
+                        length,
+                    )
```

Comment:
🗺️ The 1.0 container types have caching stubbed out for trait conformance, but it needs to be implemented. I'll open an issue for this.